detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>[minor] why define "early_termination_policy" and "max_total_runs" here? We can just set them directly in the HyperDriveConfig constructor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Changed.